src/main/resources/openapi/hearings.yaml

```diff
@@ -239,6 +286,49 @@ components:
         status:
           type: string
           example: "Available"
+    annotation:
+      type: object
+      properties:
+        annotation_id:
+          type: integer
+          example: 1
+        hearing_id:
+          type: integer
+          example: 2
+        hearing_date:
+          type: string
+          format: date
+          example: 2022-05-20
+        annotation_ts:
+          type: string
+          format: date-time
+          example: 2023-06-20T10:00:00Z
+        annotation_text:
+          type: string
+          example: "A summary notes of this annotation..."
+        annotation_documents:
+          type: array
+          items:
+            $ref: '#/components/schemas/annotation_document'
+    annotation_document:
+      type: object
+      properties:
+        annotation_document_id:
+          type: integer
+          example: 1
+        file_name:
+          type: string
+          example: "Annotation.doc"
+        file_type:
+          type: string
+          example: "DOC"
+        uploaded_by:
+          type: string
+          example: "Mr User McUserFace"
+        uploaded_ts:
```

Comment:
Maybe too late if the interface is already agreed, but consider changing to `uploaded_at` which would be better aligned with https://hmcts.github.io/restful-api-standards/#235.

Reply:
i've asked Dave, but he thinks uploaded_ts is fine, as its obvious its not a boolean, and we are already using the _ts naming convention elsewhere